sycl/test-e2e/GroupAlgorithm/reduce_sycl2020.cpp

```diff
@@ -1,6 +1,8 @@
 // RUN: %{build} -fsycl-device-code-split=per_kernel -I . -o %t.out
 // RUN: %{run} %t.out
 
+// Disabled on PVC without igc-dev due to timeout.                                                                      // https://github.com/intel/llvm/issues/14826                                                                           // UNSUPPORTED: arch-intel_gpu_pvc && !igc-dev
```

Comment:
guessing it's a typo that these comments are all on one line rather than individual lines per `//` comment